README.md

```diff
@@ -66,30 +66,30 @@ For a brief introduction to configuring the syslog-ng application, see the [quic
 
 ## Features
 
-  * receive and send [RFC3164](https://tools.ietf.org/html/rfc3164)
+  * Receive and send [RFC3164](https://tools.ietf.org/html/rfc3164)
     and [RFC5424](https://tools.ietf.org/html/rfc5424) style syslog
     messages
-  * work with any kind of unstructured data
-  * receive and send [JSON](http://json.org/) formatted messages
-  * classify and structure logs with built-in parsers (csv-parser(),
-    db-parser(), kv-parser(), ...)
-  * normalize, crunch, and process logs as they flow through the system
-  * hand over messages for further processing using message queues (like
-    [AMQP](http://www.amqp.org/)), files or databases (like
-    [PostgreSQL](http://www.postgresql.org/) or
-    [MongoDB](http://www.mongodb.org/)), and
-  * forward log messages to big data tools like [Elasticsearch](https://www.elastic.co/),
+  * Receive and send [JSON](http://json.org/) formatted messages
+  * Work with any kind of unstructured data
+  * Classify and structure logs using built-in parsers (csv-parser(),
+    db-parser(), kv-parser(), etc.)
+  * Normalize, crunch, and process logs as they flow through the system
+  * Hand over logs for further processing using files, message queues (like
+    [AMQP](http://www.amqp.org/)), or databases (like
+    [PostgreSQL](http://www.postgresql.org/) o
```

Comment:
I would prefer to stick with `or`.

Reply:
Just to be clear, this is talking about the `o` at the end? (Which is certainly a typo and I have fixed in the current commit.)

Reply:
Yes, thanks.